provider-profile-service/src/main/java/team/proximity/provider_profile_service/payment_method/PaymentMethodRequest.java

```diff
@@ -36,6 +36,10 @@ public record PaymentMethodRequest(
        String lastName,
 
        @Email(message = "Invalid email format.")
+        @Pattern(
```

Comment:
Do you really need this?
isn't `@Email` validator checking that?

Reply:
The validation was not being enforced properly. To address this, additional patterns can be applied to strengthen the enforcement, but will  look into that